keps/sig-apps/2599-minreadyseconds-for-statefulsets/README.md

```diff
@@ -589,6 +604,10 @@ details). For now, we leave it here.
 
 ###### How does this feature react if the API server and/or etcd is unavailable?
 
+ This feature will not work if the API server or etcd is unavailable as the controller-manager won't be even able get events or updates for StatefulSets. 
+ If the API server and/or etcd is unavailable during the mid-rollout, the featuregate may be enabled but it won't have any effect on the StatefulSet as
+ the controller-manager cannot communicate with the API server
```

Comment:
This also doesn't sound right; if the field is present on the object that the controller manager is working on, why would this stop working?

Reply:
It needs to do an update on the status subresource of the STS object which is not possible without communicating with kube-apiserver